src/current/v24.3/wal-failover.md

```diff
@@ -0,0 +1,495 @@
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+NAME    MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
+loop0     7:0    0  63.9M  1 loop /snap/core20/2182
+loop1     7:1    0 354.3M  1 loop /snap/google-cloud-cli/223
+loop2     7:2    0    87M  1 loop /snap/lxd/27428
+loop3     7:3    0  39.1M  1 loop /snap/snapd/21184
+sda       8:0    0    10G  0 disk
+├─sda1    8:1    0   9.9G  0 part /
+├─sda14   8:14   0     4M  0 part
+└─sda15   8:15   0   106M  0 part /boot/efi
+nvme0n1 259:0    0   375G  0 disk /mnt/data1
+nvme0n2 259:1    0   375G  0 disk /mnt/data2
+~~~
+
+Here we can see the path to `data2` volume is `/mnt/data2`. For each node, create the directory `/mnt/data2/cockroach` on the side disk volume where the WAL failover data can be written. `/mnt/data1` will be used as the single user data store for the cluster.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+mkdir /mnt/data2/cockroach
+~~~
+
+<a name="5-sidecar-start"></a>
+
+#### 5. Starting the cluster with WAL failover enabled
+
+To enable WAL failover when we start the cluster,  we can either pass the [`--wal-failover=path={ path-to-my-side-disk-for-wal-failover }` to `cockroach start`]({% link {{ page.version.version }}/cockroach-start.md%}#enable-wal-failover)
+
+ *OR*
+
+set [the environment variable `COCKROACH_WAL_FAILOVER=path={ path-to-my-side-disk-for-wal-failover }`]({% link {{ page.version.version }}/cockroach-start.md%}#enable-wal-failover) before starting the cluster.
+
+Additionally, we must set the value of the environment variable `COCKROACH_ENGINE_MAX_SYNC_DURATION_DEFAULT` to `40s`. This variable controls the duration of stall when CockroachDB crashes the node.
+
+We must also [configure logs]({% link {{ page.version.version }}/configure-logs.md %}) by passing in the `logs.yaml` file that we configured in [Step 3](#3-sidecar-configure-logs).
+
+Replicate the shell commands below on each node using your preferred deployment model. Be sure to edit the values of the flags passed to [`cockroach start`]({% link {{ page.version.version }}/cockroach-start.md %}) as needed for your environment.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+$ export COCKROACH_LOG_MAX_SYNC_DURATION=40s
+$ export COCKROACH_ENGINE_MAX_SYNC_DURATION_DEFAULT=40s
+$ cockroach start --certs-dir certs --listen-addr=:26257 --http-addr=:26258 --advertise-addr=10.150.0.54:26257 --join=34.48.145.131:26257 --store path=/mnt/data1/cockroach,attrs=store1:node1:node1store1 --cache=25% --locality=cloud=gce,region=us-east4,zone=us-east4-a,rack=0 --log-config-file=logs.yaml --wal-failover=path=/mnt/data2/cockroach
+~~~
+
+Notice the flags passed to `cockroach start`:
+
+- `--wal-failover`'s value is `path=/mnt/data2/cockroach` since that is the single failover disk
+- `--log-config-file` is `logs.yaml`
+- and the data store (store 1) path in `--store` is `/mnt/data1/cockroach`
+
+<a name="multi-store-config"></a>
+
+### Provisioning a multi-store cluster for WAL failover
```

Comment:
nit: my personal preference would be for us to highlight the multi-store configuration first. It's a bit easier and makes more sense economically since the secondary is not dormant most of the time.

Reply:
that makes sense, i have moved the multi-store configuration to be first in the doc